lms/services/canvas_api/_basic.py

```diff
@@ -95,7 +94,7 @@ def _send_prepared(self, request, schema, request_depth=1):
         result = None
         try:
             result = schema(response).parse()
-        except ValidationError as err:
+        except ExternalRequestError as err:
```

Comment:
`schema` here is an instance of one of the `RequestsResponseSchema` subclass defined in [`services/canvas_api/client.py`](https://github.com/hypothesis/lms/blob/master/lms/services/canvas_api/client.py), or I think it can also be `OAuthTokenResponseSchema` which is also a `RequestsResponseSchema` base class. `RequestsResponseSchema` now raises `ExternalRequestError` rather than `ValidationError` so this just needs to catch that instead